src/app_main.c

```diff
@@ -56,6 +57,8 @@ void app_main() {
     // Reset context
     explicit_bzero(&G_context, sizeof(G_context));
 
+    reset_dynamic_token_storage();
+
```

Comment:
Why do we need a specific reset function as Dynamic Token information is stored in RAM?

Reply:
True it's confusing, changed to init_dynamic_token_storage